docs/devguide.md

```diff
@@ -0,0 +1,234 @@
+# Development guide
+
+
+
+## Overview
+
+Kubestone is implemented as a [Kubernetes Operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) in [Go language](https://golang.org) with the help of [Kubebuilder](https://kubebuilder.io). 
+
+As with all operators, the manager deployment ties together multiple controllers, whose responsibility is to reach the desired state set by Custom Resources. In case of Kubestone the **controllers are implementing the benchmark execution logic**. Therefore, we have one controller per each supported benchmark. 
+
+
+
+!!! hint
+    If you are not familiar with Kubebuilder, it is advised to read through it's [super awesome documentation](https://book.kubebuilder.io). It contains very detailed information on how the reconcile loop should be created. This information will be essential if you would like to add support for a new benchmark in Kubestone.
+
+
+
+## Dev box setup (OS X & Linux)
+
+### Compiler tools
+
+The following software are required to build and run Kubestone:
+
+- [GoLang](https://golang.org/dl/) v1.11 or greater 
```

Comment:
Have you actually tested with v1.11? I'm pretty sure I had some issues with 1.11 on Ubuntu, so I had to upgrade to 1.12

Reply:
Yes, validated it:
```
$ go version
go version go1.11.5 darwin/amd64
$ make manager
/Users/wgershwitz/goenvs//bin/controller-gen object:headerFile=./hack/boilerplate.go.txt paths=./api/...
go fmt ./...
api/v1alpha1/doc.go
go vet ./...
go build -o bin/manager main.go
```

IIRC your issue was related to having golang 1.10, which does not support `GO111MODULES`.